Automatic Validation ended with:
> ##[error] Installer https://download.royalapps.com/RoyalTS/RoyalTSInstaller_6.01.60512.0.msi 
> ##[error] Result: Failed 
> ##vso[task.logissue type=error] Specified hash doesn't match. 
> Actual hash:   26884BBD515F4701C070BA18C0518B64055C387113D9019E4BCFD89CE8A43A16 
> Expected hash: 58e7a7dc5eb12c3bb337de2554e0dd35f3504514715114b27d7b15b1c97bbd94

--- a/manifests/r/RoyalApps/RoyalTS/6.1.60512.0/RoyalApps.RoyalTS.installer.yaml
+++ b/manifests/r/RoyalApps/RoyalTS/6.1.60512.0/RoyalApps.RoyalTS.installer.yaml
@@ -8,7 +8,7 @@ Installers:
 - Architecture: x64
   InstallerType: wix
   InstallerUrl: https://download.royalapps.com/RoyalTS/RoyalTSInstaller_6.01.60512.0.msi
-  InstallerSha256: 58E7A7DC5EB12C3BB337DE2554E0DD35F3504514715114B27D7B15B1C97BBD94
+  InstallerSha256: 26884BBD515F4701C070BA18C0518B64055C387113D9019E4BCFD89CE8A43A16
   ProductCode: '{2408F291-807B-43A8-9731-61C35F00DE1B}'
 ManifestType: installer
 ManifestVersion: 1.4.0

--- a/manifests/r/RoyalApps/RoyalTS/6.1.60512.0/RoyalApps.RoyalTS.installer.yaml
+++ b/manifests/r/RoyalApps/RoyalTS/6.1.60512.0/RoyalApps.RoyalTS.installer.yaml
@@ -4,6 +4,7 @@
 PackageIdentifier: RoyalApps.RoyalTS
 PackageVersion: 6.1.60512.0
 MinimumOSVersion: 10.0.0.0
+Scope: machine
 Installers:
 - Architecture: x64
   InstallerType: wix

--- a/manifests/r/RoyalApps/RoyalTS/6.1.60512.0/RoyalApps.RoyalTS.locale.en-US.yaml
+++ b/manifests/r/RoyalApps/RoyalTS/6.1.60512.0/RoyalApps.RoyalTS.locale.en-US.yaml
@@ -7,7 +7,7 @@ PackageLocale: en-US
 Publisher: Royal Apps GmbH
 PackageName: Royal TS
 PackageUrl: https://www.royalapps.com/ts/win/features
-License: Copyright 2021 Royal Apps GmbH
+License: Proprietary
 LicenseUrl: https://support.royalapps.com/support/solutions/articles/17000074360
 ShortDescription: Using RDP, VNC, SSH based terminals, S/FTP or web-based interfaces
 Moniker: royalts

